### PR TITLE
Ack-eliciting, not inflight

### LINKED ISSUE #2093: Clarification on Setting the Loss Detection Timer in the loss recovery draft
Setting the Loss Detection Timer section states:

    // Don't arm timer if there are no retransmittable packets
    // in flight.
    if (bytes_in_flight == 0):
      loss_detection_timer.cancel()
      return

If a QUIC sender sends multiple retransmittable packets and some ACK/PADDING packets and only retransmittable packets are ACKed, it leaves the bytes_in_flight > 0 but no retransmittable packets inflight. In this case, Loss Detection Timer need not be scheduled as there is not retransmittable packets inflight but the above check results in scheduling the timer again.

if (bytes_in_flight == 0): check does not seem enough by itself to decide if there are outstanding retransmittable packets. Do we need to update that section?

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #2093

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -792,7 +792,7 @@ Pseudocode for SetLossDetectionTimer follows:
  SetLossDetectionTimer():
     // Don't arm timer if there are no ack-eliciting packets
     // in flight.
-    if (bytes_in_flight == 0):
+    if (no ack-eliciting packets oustanding):
```
[the remark this answers is not in the document]
Comment (janaiyengar) [reply draft-ietf-quic-recovery.md]: we use it below, which is why I used it here. I could change them both to "in flight" or leave them at outstanding

Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
    if (no ack-eliciting packets outstanding):
```

This is a mistake I often make too.

[the remark this answers is not in the document]
Comment (ianswett) [reply draft-ietf-quic-recovery.md]: I sent #2120 to remove existing uses of outstanding.

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -792,7 +792,7 @@ Pseudocode for SetLossDetectionTimer follows:
  SetLossDetectionTimer():
     // Don't arm timer if there are no ack-eliciting packets
     // in flight.
-    if (bytes_in_flight == 0):
+    if (no ack-eliciting packets outstanding):
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]:
```suggestion
     if (no ack-eliciting packets in flight):
```

**Pull request merged**